README.MD

```diff
@@ -0,0 +1,10 @@
+# README.MD - epipipeline v2.0.11
+
+This repository contains code that can be used to standardise linelists and summaries.
+
+The standards currently used are:
+
+    1. Linelists: [YAML](https://github.com/dsih-artpark/data_documentation/documentation/EP/EP0005DS0014-KA_Dengue_LL/datadictionary.yaml), [MD]([YAML](https://github.com/dsih-artpark/data_documentation/documentation/EP/EP0005DS0014-KA_Dengue_LL/datadictionary.MD))
```

Comment:
1. location.admin.hierarchy (revenue/ulb) may be redundant as we have explicit variable names for each geographic unit
2. event.test.value considering an edge case where test results vary by test type (e.g., NS1 is positive but IgM is negative), in which case we need event.test1.value, event.test2.value, etc.